interfaces/Config/templates/config_switches.tmpl

```diff
@@ -324,11 +324,6 @@
                         <span class="desc">$T('explain-rating_enable')</span>
```

Comment:
I'm not sure why this option needs to go.
Even though someone uses an indexer, he/she may not necessarily want to provide rating feedback.